numpy/core/src/common/npy_cpu_features.c

```diff
@@ -218,27 +241,29 @@ npy__cpu_validate_baseline(void)
 }
 
 static int
-npy__cpu_try_disable_env(void)
-{
-    char *disenv = getenv("NPY_DISABLE_CPU_FEATURES");
-    if (disenv == NULL || disenv[0] == 0) {
-        return 0;
+npy__cpu_check_env(int disable, char* env){
+
+    char action[8];
+    if (disable){
+        strcpy(action, "DISABLE");
+    } else{
+        strcpy(action, "ENABLE");
```

Comment:
```suggestion
        static const char *names[] = {
        "enable", "disable",
        "NPY_ENABLE_CPU_FEATURES", "NPY_DISABLE_CPU_FEATURES",
        "During parsing environment variable: 'NPY_ENABLE_CPU_FEATURES':\n",
        "During parsing environment variable: 'NPY_DISABLE_CPU_FEATURES':\n"
    };
    disable = disable ? 1 : 0;
    const char *env_name = names[disable + 2];
    const char *err_head = names[disable + 4];
    const char *act_name = names[disable];
```
How about offset over static storage?